PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
That looks right to me, i.e. `Sequence`.

I know for some types it's a little ambiguous; like, when should I only use `Iterable` instead of `Sequence`? I see for the former we have `__abstractmethods__ = frozenset({'__getitem__', '__len__'})`, and then for the latter `__abstractmethods__ = frozenset({'__iter__'})`, but I have yet to see an implementation of an interable in Python that doesn't use `__getitem__` with `__iter__` to retrieve values.